packages/sanity/src/desk/comments/src/context/comments/CommentsProvider.tsx

```diff
@@ -43,66 +42,11 @@ export interface CommentsProviderProps {
   documentType: string
 }
 
-const EMPTY_COMMENTS = {
-  data: EMPTY_COMMENTS_DATA,
-  error: null,
-  loading: false,
-}
-
-const EMPTY_MENTION_OPTIONS = {
-  data: [],
-  error: null,
-  loading: false,
-}
-
-const noop = async () => {
-  await Promise.resolve()
-}
-
-const noopOperation = {
-  execute: noop,
-}
-
-const COMMENTS_DISABLED_CONTEXT: CommentsContextValue = {
-  comments: EMPTY_COMMENTS,
-  create: noopOperation,
-  edit: noopOperation,
-  getComment: () => undefined,
-  isRunningSetup: false,
-  mentionOptions: EMPTY_MENTION_OPTIONS,
-  remove: noopOperation,
-  setStatus: noop,
-  status: 'open',
-  update: noopOperation,
-}
-
 /**
  * @beta
- * @hidden
  */
 export const CommentsProvider = memo(function CommentsProvider(props: CommentsProviderProps) {
   const {children, documentId, documentType} = props
-
-  const {isEnabled} = useCommentsEnabled({
-    documentId,
-    documentType,
-  })
-
-  if (!isEnabled) {
```

Comment:
This can be removed now since we make sure to only render the `CommentField` component and the `CommentsDocumentInspector` (which uses `useComments`) if comments is enabled by looking at the enabled value from `CommentsEnabledContext.